Olha_Lesiuk/HW_10/task_1.py

```diff
@@ -0,0 +1,29 @@
+class Toyota:
```

Comment:
not okay. It should be a class Company. And I want to create any company I want. In your implementation it always be Toyota

Reply:
Fixed